docs/overview/about.md

```diff
@@ -0,0 +1,88 @@
+# About Vald <!-- omit in toc -->
+
+This document gives an overview of what is Vald and what you can do with Vald. 
+
+## Table of Contents <!-- omit in toc -->
+
+- [What is Vald?](#what-is-vald)
+  - [What Vald can do?](#what-vald-can-do)
+  - [Use cases](#use-cases)
+- [Why Vald?](#why-vald)
+- [How does Vald work?](#how-does-vald-work)
+- [Try Vald](#try-vald)
+
+### What is Vald?
+
+<!-- copied from README.md-->
+Vald is a highly scalable distributed fast approximate nearest neighbor dense vector search engine.
+
+Vald is designed and implemented based on Cloud-Native architecture.
+
+It uses the fastest ANN Algorithm [NGT](https://github.com/yahoojapan/NGT) to search neighbors.
+
+Vald has automatic vector indexing and index backup, and horizontal scaling which made for searching from billions of feature vector data.
+
+Vald is easy to use, feature-rich and highly customizable as you needed.
+
+#### What Vald can do?
+
+<!-- copied from README.md-->
+- Asynchronize Auto Indexing
+    - Usually the graph requires locking during indexing, which causes stop-the-world. But Vald uses distributed index graphs so it continues to work during indexing.
+
+- Customizable Ingress/Egress Filtering
+    - Vald implements it's own highly customizable Ingress/Egress filter.
+    - Which can be configured to fit the gRPC interface.
+        - Ingress Filter: Ability to Vectorize through filter on request.
+        - Egress Filter: rerank or filter the searching result with your own algorithm.
+
+- Cloud-native based vector searching engine
+    - Horizontal scalable on memory and CPU for your demand.
+
+- Auto Indexing Backup
+    - Vald has an auto index backup feature using MySQL + Redis or Cassandra which enables disaster recovery.
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
to tell the truth, Redis is not used for index backup, just for storing meta KV. :rofl: